src/intl/messages/de.json

```
@@ -291,6 +291,116 @@
 			"newEmail": "Neue E-Mail",
 			"type": "Typ",
 			"belowToConfirm": "unten zur Bestätigung",
+			"Notifications": {
+				"notificationChannels": "Benachrichtigungskanäle",
+				"configureNotificationChannels": "Bitte wählen Sie die sozialen Medien aus, über die Sie Benachrichtigungen erhalten möchten:",
```

Biome (2.1.2)

[error] The key notificationChannels was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)